GAD 6 - Verbessertes Sortieren/tests/simplesort/DualPivotFinderTester.java

```diff
@@ -271,8 +272,17 @@ public void getRandomPivots() {
     @Test
     void medianPivotFront() {
         int[] arr;
+        int[] returnedElements;
+
         arr = new int[] {213, 180, 93, 90, 46};
-        assertArrayEquals(new int[]{3, 2}, DualPivotFinder.getMedianPivotFront(5).findPivot(arr, 2, 3));
+
+        //Note: I know it's a lazy fix, but it makes it more understadable for debugging
+        returnedElements = DualPivotFinder.getMedianPivotFront(5).findPivot(arr, 2, 3);
+        if (returnedElements[0] == 2 && returnedElements[1] == 3) {
+        } else if (returnedElements[0] == 3 && returnedElements[1] == 2) {
+        } else
+            fail("You did not return correct pivots for the array " + Arrays.toString(arr) + "\n" + "The correct pivots were either [0, 1] or [1, 0] but you returned " + Arrays.toString(returnedElements));
+
         arr = new int[] {138, 68, 111, 255, 33};
         assertArrayEquals(new int[]{1, 2}, DualPivotFinder.getMedianPivotFront(5).findPivot(arr, 1, 3));
```

Comment:
This test also fits my issue but has not been fixed. There are a lot of lines like this